### ISSUE TITLE
Training Time

### ISSUE THREAD

**Issue by Huster-Hq** [hkchengrex/Cutie#72]
Hi, do you only include 'main_training part' in the '48 hours' here?

Please ask the total number of frames in your training data set, as well as your total number of training iterations is 125000 and batch_size is 16? (followed your train_config.yaml)

![image](https://github.com/hkchengrex/Cutie/assets/100923014/d526e464-e25a-4713-87f6-cd17fe4e0bbe)

**Comment by hkchengrex** [comment]
It is for the entire training process.

I don't have the total number of training frames off the top of my head, but you can just count from the training dataset.

The default configuration in the code is correct.

**Comment by Huster-Hq** [comment]
Thanks!

I currently face a problem:  the training time is too long when I run train.py on our custom dataset (two 3090 GPUs).

My custom dataset: 19544 frames in total, and I set the 'iteration=125000', and 'batch_size=14'.

The following figures are my training record and GPU. Is this reasonable?
![image](https://github.com/hkchengrex/Cutie/assets/100923014/a45fdcb3-1512-4af0-b2d8-05e125fddd2b)
![image](https://github.com/hkchengrex/Cutie/assets/100923014/7fe68cc5-59ef-4a29-b452-64edc0bc906c)

**Comment by hkchengrex** [comment]
1. The first printed "time" is not reliable as it includes warm-up and initialization.
2. In general, we want a high GPU utilization (see https://github.com/hkchengrex/Cutie/issues/58#issuecomment-2053740579) -- your GPUs are sitting at 18C and 29C so I don't think they are working hard enough unless you have really good cooling. This usually indicates a bottleneck in CPU/disk IO.

**Comment by Huster-Hq** [comment]
At the later 'time', the 'avg_time' is still 12-13.

![image](https://github.com/hkchengrex/Cutie/assets/100923014/fa9bdf93-f447-4a83-89f7-6f12f9d26377)
![image](https://github.com/hkchengrex/Cutie/assets/100923014/ffd9d6a9-586f-499f-ad34-ea6260e324dd)

**Comment by hkchengrex** [comment]
Your CPUs and GPUs are not being used as much as they should be. Potentially related: https://github.com/pytorch/pytorch/issues/99625

**Comment by hkchengrex** [comment]
Please feel free to re-open if there are any updates.
